### ISSUE TITLE
Add support for InetAddressIPv6

### ISSUE THREAD

**Issue by kbabioch** [drkjam/netaddr#170]
InetAddressIPv6 as used in SNMP (RFC4001) deals with IPv6 addresses in the following way:

`fe:80:00:00:00:00:00:00:c2:ee:fb:ff:12:ed:12:96`

The details can be found [here](https://www.ietf.org/rfc/rfc4001.txt):

    InetAddressIPv6 ::= TEXTUAL-CONVENTION
        DISPLAY-HINT "2x:2x:2x:2x:2x:2x:2x:2x"
        STATUS       current
        DESCRIPTION
            "Represents an IPv6 network address:

               Octets   Contents         Encoding
                1-16    IPv6 address     network-byte order

             The corresponding InetAddressType value is ipv6(2).

             This textual convention SHOULD NOT be used directly in object
             definitions, as it restricts addresses to a specific format.
             However, if it is used, it MAY be used either on its own or in
             conjunction with InetAddressType, as a pair."
        SYNTAX       OCTET STRING (SIZE (16))

Basically there are 16 octets encoded as hex divided by `:`.
